src/Icon.svelte

```diff
@@ -70,6 +74,55 @@
     <path
       d="M7 12a2 2 0 1 1-4.001-.001A2 2 0 0 1 7 12zm12-2a2 2 0 1 0 .001
     4.001A2 2 0 0 0 19 10zm-7 0a2 2 0 1 0 .001 4.001A2 2 0 0 0 12 10z" />
+  {:else if name === "chevron-down"}
+    <path
+      fill-rule="evenodd"
+      clip-rule="evenodd"
+      d="m 16.353549 9.8535548 c 0.19527 0.1952622 0.19527 0.5118432 0
+      0.7071032 l -3.29289 3.2929 c -0.58579 0.58578 -1.53553 0.58578
+      -2.12132 0 l -3.292894 -3.2929 c -0.195262 -0.19526 -0.195262
+      -0.511841 0 -0.7071042 0.195263 -0.195262 0.511844 -0.195262
+      0.707104 0 l 3.2929 3.2928942 c 0.19526 0.19526 0.51184 0.19526
+      0.7071 0 l 3.2929 -3.2928932 c 0.19526 -0.195262 0.51184
+      -0.195262 0.7071 0 z"
+      fill="currentColor" />
```

Comment:
I think we can remove the fill here, since we set it in `<svg>` further up.

Reply:
2f20d210